packages/rhi-webgl/src/WebGLEngine.ts

```diff
@@ -9,15 +10,19 @@ export class WebGLEngine extends Engine {
   /**
    * Create an engine suitable for the WebGL platform.
    * @param canvas - Native web canvas
+   * @param physicsEngine - Physics Engine
    * @param webGLRendererOptions - WebGL renderer options
    */
-  constructor(canvas: string | HTMLCanvasElement | OffscreenCanvas, webGLRendererOptions?: WebGLRendererOptions) {
+  constructor(
+    canvas: string | HTMLCanvasElement | OffscreenCanvas,
+    physicsEngine?: IPhysics,
```

Comment:
rename param name to `physics?: IPhysics`.

Reply:
done